pkg/tsdb/grafana-pyroscope-datasource/instance.go

```diff
@@ -98,21 +90,7 @@ func (d *PhlareDatasource) profileTypes(ctx context.Context, req *backend.CallRe
 }
 
 func (d *PhlareDatasource) labelNames(ctx context.Context, req *backend.CallResourceRequest, sender backend.CallResourceResponseSender) error {
-	u, err := url.Parse(req.URL)
-	if err != nil {
-		return err
-	}
-	query := u.Query()
-	start, err := strconv.ParseInt(query["start"][0], 10, 64)
-	if err != nil {
-		return err
-	}
-	end, err := strconv.ParseInt(query["end"][0], 10, 64)
-	if err != nil {
-		return err
-	}
-
-	res, err := d.client.LabelNames(ctx, query["query"][0], start, end)
```

Comment:
Curious about this. Why remove `query`, `start` and `end`? Is it because we were not actually using them in the datasource?

Reply:
Because the phlare client (which is also new pyroscope client) does not use it so we would just have unused args there. We actually send it from frontend so it will be easy to add them again once pyroscope supports it.